error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/IsInstanceLambdaUsage.java

```diff
@@ -24,7 +24,7 @@
  * A {@link BugChecker} that flags lambda expressions that can be replaced with a method reference
  * of the form {@code T.class::isInstance}.
  *
- * @see MethodReferenceUsage
+ * <p>See the `MethodReferenceUsage` check in the `error-prone-experimental` module.
```

Comment:
Here and below: instead of backticks we should use `{@code ..}`. But since there's a corresponding `XXX` comment: let's drop.